data/rulesets.js

```diff
@@ -256,7 +262,8 @@ exports.BattleFormats = {
 		},
 	},
 	hoennpokedex: {
-		effectType: 'Rule',
+		effectType: 'Banlist',
```

Comment:
There's currently no difference in how they are handled. The new distinction allows the feature to limit ruleset changes to those that have any effect.